xarray/core/common.py

```diff
@@ -243,6 +244,8 @@ def get_squeeze_dims(xarray_obj, dim, axis=None):
 class DataWithCoords(SupportsArithmetic, AttrAccessMixin):
     """Shared base class for Dataset and DataArray."""
 
+    _rolling_exp_cls = RollingExp
```

Comment:
I don't think we need this class attribute any longer, but leaving for the sake of conformity unless we have consensus